guides/Makefile

```diff
@@ -1,37 +1,89 @@
 SHELL := /bin/bash
-ALL_DIRS = $(shell ls -d doc-*)
-GUIDES-HTML = $(ALL_DIRS)
+ALL_DIRS = $(wildcard doc-*)
+ALL_GUIDES = $(patsubst doc-%,%,$(ALL_DIRS))
+BUILDS = foreman-deb foreman-el katello satellite orcharhino
+BUILD_DIR := build
+COMMON_DIR := common
+DEPENDENCIES := $(shell find $(COMMON_DIR) -type f -name \*.adoc)
+DATE_MDY := $(shell LC_ALL=C date "+%b %d %Y")
+DATE_MY := $(shell LC_ALL=C date "+%b %Y")
 
-.PHONY: all html pdf linkchecker all-html all-pdf clean all-clean linkchecker-tryer $(GUIDES-HTML)
+HTMLS := $(BUILDS:%=%.html)
+PDFS := $(BUILDS:%=%.pdf)
+GUIDE_BUILDS := $(addprefix $(BUILD_DIR)/,$(ALL_GUIDES))
+
+GUIDES-HTML := $(foreach guide,$(GUIDE_BUILDS),$(addprefix $(guide)/,$(HTMLS)))
+GUIDES-PDF := $(foreach guide,$(GUIDE_BUILDS),$(addprefix $(guide)/,$(PDFS)))
+
+.PHONY: all html pdf clean linkchecker linkchecker-tryer
 
 all: html
 
-html: all-html
+html: $(GUIDES-HTML)
 
-pdf: all-pdf
+pdf: $(GUIDES-PDF)
 
-clean: all-clean
+clean:
+	rm -rf $(BUILD_DIR)
 
 linkchecker:
-	find build -type f -name index\*html | xargs linkchecker -r1 -f common/linkchecker.ini --check-extern
+	find $(BUILD_DIR) -type f -name \*.html | xargs linkchecker -r1 -f common/linkchecker.ini --check-extern
```

Comment:
Note I changed `index-$build` to `$build`. However, I haven't modified the web parts yet.